### PR TITLE
tidy errors

### PR DESCRIPTION
**Pull request by MirijaHa**
As we noted in dryrun that current error messages are not particularly helpful I am adjusting the most likely errors that will require user action to the admittedly more informative tidyverse convention.
WIll add a follow up PR for input argument validations when @maurolepore s suggestion here  https://github.com/2DegreesInvesting/r2dii.climate.stress.test/pull/189 is implemented, as this will simplify adapting the errors for input args validation.

### CODE REVIEW

**Review by jacobvjk — APPROVED** [review]
lgtm

File: R/utils.R [hunk 1]
```diff
@@ -66,14 +66,22 @@ path_dropbox_2dii <- function(...) {
...
-  stopifnot(valid_file_path)
+
+  if (!valid_file_path) {
+    rlang::abort(c(
+      "Detected invalid file path.",
+      x = glue::glue("Invalid file path: {file.path(path)}."),
+      i = "Did you set path to data correctly?."
+    ))
+  }
+  invisible()
```
Comment (maurolepore) [inline R/utils.R:84]: This should be path.

> If a function is called primarily for its side-effects, it should invisibly return a useful output. If there’s no obvious output, return the first argument. This makes it possible to use the function with in a pipeline.

-- https://design.tidyverse.org/out-invisible.html


```r
"/a/path" %>%
  validate_file_exists() %>%
  do_something_else_with_the_path()
```

File: R/utils.R [hunk 2]
```diff
@@ -66,14 +66,22 @@ path_dropbox_2dii <- function(...) {
...
+#' @return NULL
 #'
 #' @export
 validate_file_exists <- function(path) {
   valid_file_path <- file.exists(file.path(path))
-  stopifnot(valid_file_path)
+
+  if (!valid_file_path) {
+    rlang::abort(c(
+      "Detected invalid file path.",
```
Comment (maurolepore) [inline R/utils.R:79]: Nit:

> If the cause of the problem is clear, use “must”:
-- https://style.tidyverse.org/error-messages.html#problem-statement

Suggested:

"`path` must point to an existing file."

**Review by maurolepore — COMMENTED** [review]
Thanks! I make small nit suggestions.

**Pull request merged**